lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -20,6 +20,12 @@ fabric_forwarding:
   get_value: '/^feature fabric forwarding$/'
   set_value: "feature fabric forwarding"
 
+itd:
+  _exclude: [N3k, N5k, N6k]
```

Comment:
Checked with Deepak and we decided to not support n8k until its code comes close to n7k/n9k.